tests/acceptance/features/bootstrap/RevaContext.php

```diff
@@ -1,59 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
This was a "placeholder" when we got the local acceptance tests started. It is not needed any more.